Android/PushTutorial/push-tutorial-one/build.gradle

```diff
@@ -0,0 +1,46 @@
+apply plugin: 'com.android.application'
+
+android {
+    compileSdkVersion 28
+    defaultConfig {
+        applicationId "io.ably.example.androidpushexample"
```

Comment:
as commented, pls assign new ids - eg `io.ably.tutorial.push-tutorial-one` etc

Reply:
I have changed the applicationId to `io.ably.tutorial.push_tutorial_one` and `io.ably.tutorial.push_tutorial_two`